### DISCUSSION TITLE
Search in static a JSON

### DISCUSSION THREAD

**Discussion by RonOrgad** [schibsted/jslt#364] [Q&A]
hey, 
i have a small static json for example 
{
    "country": {
        "usa": {
            "new york": 445,
            "los_angeles": 678
        },
        "canada": {
            "toronto": 550,
            "vancouver": 320
        },
        "uk": {
            "london": 890,
            "manchester": 270
        },
        "australia": {
            "sydney": 360,
            "melbourne": 410
        }
    }
}
and i have my input JSON ( i am using JSLT on apache NIFI) 
In my input json i have country and city keys. I want to create a new key per json with the city code according to the static json 
For example for the input :
{
    "country" : "usa" , 
    "city" : "new york"
}
i want the output will be : 
{
    "country" : "usa" , 
    "city" : "new york", 
    "city_code" : "445"
}
I am using the processor JSLTTransormJSON and i defined the static json as a variable. 
I would be happy to help with the continuation 
thanks

**Comment by larsga** [comment]
Have you tried writing some JSLT? What is the problem?

**Reply by RonOrgad** [reply]
Yes i have tried, and i cant do 
let country = .country
and after that 
let city = .city
it says Encountered ""=""
if i keep one of them it works fine

**Comment by larsga** [comment]
Try putting a parenthesis around `.country`. Also, it's better if you give complete examples. Without those it's not really possible to know what's wrong.

**Comment by RonOrgad** [comment]
this is my jslt 
{
  "country" :  .country,
  "city" :  .city,
  "city_code" : $static_json.country[.country][.city]
}
i tried to do something like that but it seems that i cant do it - [.country][.city] because i no city_code
but when i do something like 
  "city_code" : $static_json.country or city_code" : $static_json.country."usa" i do get result.

**Comment by larsga** [comment]
The part that doesn't work isn't actually JSLT code, so it will never work. I don't even know what it means.

But what you've written here will give you a single object with three keys. That's not what you want. Look at the tutorial, and look at `for` loops.
